README.md

```diff
@@ -1,2 +1,26 @@
 # Pheromones
-A repo that regroup SwarmUS communication messages with protobuf
+A repo that regroup SwarmUS communication messages with protobuf.
+
+
+## Requirements
+
+* `protoc` to compile .proto files
+* python libraries for protobuf to generate the files
+
+On ubuntu 20.04 you can install it using `apt install protobuf-compiler python3-protobuf`
```

Comment:
No need for `nanopb`? Or is that already included in some of these dependencies?

Reply:
It is included by CMake in the build

Reply:
Exaclty, and you can pas NANOPB_PATH if you want to pass the lib from your package manager. (Though we need to build it on embedded anyway)